lua/entities/gmod_wire_expression2/core/player.lua

```diff
@@ -21,6 +21,12 @@ end)
 
 __e2setcost(5) -- temporary
 
+e2function steam entity:getUserGroup()
```

Comment:
This returns a *what*?

Reply:
went braindead for a second i guess, fixed it